src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -144,12 +144,13 @@ FILE* jitstdout()
 }
 
 // Like printf/logf, but only outputs to jitstdout -- skips call back into EE.
-void jitprintf(const char* fmt, ...)
+int jitprintf(const char* fmt, ...)
```

Comment:
Why is this change needed?

Reply:
Inside the `emitDispInsName` in the `emitriscv64.cpp` file the return value of the `printf` function has been used in [some places](https://github.com/dotnet/runtime/blob/413efe748ac4c5daabf087ed8ed0d59c511e5cf2/src/coreclr/jit/emitriscv64.cpp#L3252). When I've allowed this function to be compiled in the release mode I've found out that the code wasn't compiling due to the void type returned by the `printf`. Upon inspection I thought that changing signature of this function would be the easiest fix